exercises/triangle/example.js

```diff
@@ -22,8 +22,10 @@ function Triangle(a, b, c) {
   };
 
   this.violatesInequality = function () {
-    var a = this.sides[0], b = this.sides[1], c = this.sides[2];
-    return (a + b < c) || (a + c < b) || (b + c < a);
+    var sideA = this.sides[0];
```

Comment:
When triangle is created, `a`, `b` and `c` params are assigned to `this.sides`. Is there really a need to *extract* them into separate variables? In fact, would `return (a + b < c) || (a + c < b) || (b + c < a);` work without redeclaring `a`, `b` and `c`?

Just asking.

Reply:
@rchavarria You are absolutely correct.  I will alter the code.